tests/api-watch/test_expansion_lst.ml

```diff
@@ -0,0 +1,97 @@
+open Api_watch
```

Comment:
For unit tests we don't need one file per function tested. One unit test file per module (as in .ml file) is enough. Can you merge the two together?